### DISCUSSION TITLE
Multiple users on one device

### DISCUSSION THREAD

**Discussion by jordanthewebdesigner** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#639] [Q&A]
Hello, I just discovered this awesome repo and set it up on my smart mirror project.

I can't seem to figure out how to get my wife's account setup too so she can have personalized results. She has ios devices, and the prompt for her "What's my name?" shows up on my Android phone instead.

Am I missing something here, or is that setup just not possible?

**Comment by im-coder-lg** [comment]
That is called Personal Results. I think you should check the Wiki for how to enable it. If it fails, tell me here, I'll post a GH gist link.

**Reply by jordanthewebdesigner** [reply]
Oh sorry, I should have included that I am getting the notification on my phone instead of my wife's as part of the Personal Results setup process.

I think I have everything working for me, I just can't figure out how to let it recognize her voice too. I added her email on the voice API page, and it does respond with a prompt to her asking "What's my name?" but I just can't figure out how to actually add her account to the local app

**Reply by im-coder-lg** [reply]
I don't think the SDK can do that, @Melvin-Abraham might elucidate this.

**Reply by sumeshir26** [reply]
Currently, the app supports only one account one one PC, so its not possible yet☹️.
